std/random.d

```diff
@@ -1353,25 +1638,119 @@ Twister has `2 ^^ 19937 - 1` distinct states but after `seed(uint)` is
 called it can only be in one of `2 ^^ 32` distinct states regardless of
 how excellent the source of entropy is.
 */
-@property uint unpredictableSeed() @trusted nothrow @nogc
+alias unpredictableSeed = unpredictableSeedOf!uint;
+/// ditto
+@property UIntType unpredictableSeedOf(UIntType)() @nogc nothrow @trusted
```

Comment:
(as mentioned in you earlier PR)
Technically this is a new symbol and requires @andralex approval